en/flight_controller/nxp_mr_vmu_rt1176.md

```diff
@@ -0,0 +1,265 @@
+- Redundancy: 3x IMU sensors & 2x Barometer sensors on separate buses
+- Triple-redundancy domains: Completely isolated sensor domains with separate buses and separate power control
+- Newly designed vibration isolation system to filter out high frequency vibration and reduce noise to ensure accurate readings
+- 100Base-T1 2-Wire Ethernet interface for high-speed mission computer integration
+- IMUs are temperature-controlled by onboard heating resistors, allowing optimum working temperature of IMUs&#x20;
+
+### Processors & Sensors
+
+- FMU Processor: NXP i.MX RT1176
+  - 32 Bit Arm® Cortex®-M7, 1GHz
+  - 32 Bit Arm® Cortex®-M4, 400MHz secondary core
+  - 64MB external flash memory
+  - 2MB RAM
+- NXP EdgeLock SE051 hardware secure element
+  - IEC62443-4-2 certified for the applicable requirements
+  - 46 kB user memory with personalization options to go up to 104 kB
+  - Groundbreaking CC EAL6+ certified solution for IoT deployments
+  - AES and 3DES encryption and decryption
+- On-board sensors
+  - Accel/Gyro: ICM-20649 or BMI088
+  - Accel/Gyro: ICM-42688-P
+  - Accel/Gyro: ICM-42670-P
+  - Mag: BMM150
+  - Barometer: 2x BMP388
+
+### Electrical data
+
+- Voltage Ratings:
+  - Max input voltage: 6V
+  - USB Power Input: 4.75\~5.25V
+  - Servo Rail Input: 0\~36V
+- Current Ratings:
+  - `TELEM1` output current limiter: 1.5A
+  - All other port combined output current limiter: 1.5A
+
+### Mechanical data
+
+- Dimensions
+  - Flight Controller Module: 38.8 x 31.8 x 14.6mm
+  - Standard Baseboard: 50 x 96 x 16.7mm
+- Weight
+  - Flight Controller Module: 23g
+  - Standard Baseboard: 51g
+
+### Interfaces
+
+- 12 PWM servo outputs, 8 with D-SHOT
+- R/C input for Spektrum / DSM
+- Dedicated R/C input for PPM and S.Bus input
+- Dedicated analog / PWM RSSI input and S.Bus output
+- 4 general purpose serial ports:
+  - 3 with full flow control
+  - 1 with separate 1.5A current limit (Telem1)
+  - 1 with I2C and additional GPIO line for external NFC reader
+- 2 GPS ports
+  - 1 full GPS plus Safety Switch Port
+  - 1 basic GPS port
+- 1 I2C port
+- 1 Ethernet port
+  - Transformerless Applications
+  - 100Mbps
+- 1 SPI bus
+  - 2 chip select lines
+  - 2 data-ready lines
+  - 1 SPI SYNC line
+  - 1 SPI reset line
+- 3 CAN Buses for CAN peripheral (2 currently supported by PX4)
+  - CAN Bus has individual silent controls or ESC RX-MUX control
+- 2 Power input ports with SMBus
+- 1 AD & IO port
+- 2 additional analog input
+- 1 PWM/Capture input
+- 2 Dedicated debug and GPIO lines
+
+- Other Characteristics:
+  - Operating & storage temperature: -40 ~ 85°c
+
+## Where to Buy
+
+Order from [NXP](https://www.nxp.com).
+
+## Assembly/Setup
+
+Wiring is similar to the [Holybro Pixhawk 6X](../flight_controller/pixhawk6x.md#connections) and other boards that follow the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
```

Comment:
although it is similar to holybros, the key selling point for NXP is the addition of ports, so I suggest NXP invest in creating a wiring diagram

Reply:
@dk7xe This makes sense to me, though how important this is does depend on how likely this reference design is likely to match the derivatives.

@mrpollo makes a very good point though - I wasn't even aware that the new ports were "significant" from the current documentation. To me this was mostly about the reference design benefits. So if the ports are a key difference, that would be good to capture? 
I suggest that if you agree you do this as a github suggestion.

Reply:
We won't be able to make a wiring diagram like Holybro does. 
We can create a simpler block diagram